src/main/java/no/dcat/harvester/crawler/web/CrawlerRestController.java

```diff
@@ -199,12 +210,26 @@ void harvestAllDcatSources() throws InterruptedException {
             CrawlerJob job = crawlerJobFactory.createCrawlerJob(dcatSource);
             try {
                 crawler.execute(job).get();
+                updateSearch();
```

Comment:
flytt den ut av for-løkka, ellers sender du en melding pr katalog

Reply:
Fix'd